generics/verification-steps.js

```diff
@@ -1,183 +1,71 @@
+import {expect} from 'chai';
+
 module.exports = ({Then}) => {
 
     Then(/^user is taken to the "([^"]*)" page$/, async (pageURI) => {
         global.pageID = pageURI;
-
         if (!pageMap[global.pageID]) throw new Error(`Page Object with name ${pageURI} is not defined`);
-        const result = await driver.wait(until.urlMatches(pageMap[global.pageID].URL), process.env.CUCUMBER_REDIRECT_TIMEOUT * 1000, `URL never matched ${pageMap[global.pageID].URL}`)
```

Comment:
got rid of this but it was commented from you (seems like a newly refactored function), please take a look at this one.

Reply:
you should revert this one. I uncommented the remaining code in the function, as it now works